### DISCUSSION TITLE
HDR and XDR display

### DISCUSSION THREAD

**Discussion by ZSY4** [immich-app/immich#8484] [Feature Request]
Will the APP be supported to view HDR and XDR images and videos in the future?

**Comment by AngelaDMerkel** [comment]
There is some discussion about this ongoing in #3623 and #5988 but it seems unlikely that they are implemented, at least in the coming future. Vips and Sharp both don't have upstream support for Ultra HDR and it doesn't look like there is much motivation to implement from the team and @mertalev who previously implemented P3 gamut images. There are also some technical issues when converting from true HDR images to Ultra HDR that even the vips team isn't clear on how to resolve. Either way there would still be a lot of user input required to determine how they want their HDR images to be converted for display.

**Reply by ZSY4** [reply]
Thank you. Thank you very much for your reply.

**Reply by mertalev** [reply]
I think AVIF is the best option here since it natively supports HDR. I've been laying the groundwork for it: sharp now supports HDR AVIF output and we support switching image formats for both thumbnails and previews. There's still some work left to do, though, especially on mobile.

**Reply by AngelaDMerkel** [reply]
@mertalev Thanks for all the work you do. What's clear to me is that HDR formats are an absolute disaster, and everyone goes about producing them differently. Apple and Google have the most confusing implementations, imo

**Comment by kaiserkiwi** [comment]
The last comments are from a while ago. I just looked at some images in my iCloud Photos app and then afterwards the same images in the immich app (both iOS). And the difference is night and day.

Did something move regarding HDR support for immich?

**Comment by bo0tzz** [comment]
#7262
